service/collector.go

```diff
@@ -246,7 +201,11 @@ func (col *Collector) setupConfigurationComponents(ctx context.Context) error {
 	return nil
 }
 
-func (col *Collector) execute(ctx context.Context) error {
+// Run starts the collector according to the command and configuration
```

Comment:
It does not take the command into account anymore :)
```suggestion
// Run starts the collector according to the configuration
```